.ci/docker/common/install_rocm_magma.sh

```diff
@@ -2,6 +2,9 @@
 
 set -ex
 
+
```

Comment:
cc @jithunnair-amd

Reply:
Changes LGTM. I'll check the builds in a bit